cedux.h

```diff
@@ -14,15 +14,15 @@ LIST_DECLARATION(STORE_NAME##_reducer_list, STORE_NAME##_REDUCER, 32)
                                                                                                 \
 void cedux_register_##STORE_NAME##_reducer(struct STORE_NAME##_handle * p_store,                \
                                            STORE_NAME##_REDUCER reducer);                       \
-void cedux_init_##STORE_NAME(struct STORE_NAME##_handle * p_store);                             \
+struct STORE_NAME##_handle cedux_init_##STORE_NAME(void);                                       \
 void cedux_dispatch_##STORE_NAME(struct STORE_NAME##_handle * p_store, ACTION_TYPE action);     \
 bool cedux_run_##STORE_NAME(struct STORE_NAME##_handle * p_store);                              \
 struct STORE_NAME##_handle                                                                      \
 {                                                                                               \
   TREE_TYPE tree;                                                                               \
   struct STORE_NAME##_action_queue action_queue;                                                \
   struct STORE_NAME##_reducer_list reducer_list;                                                \
-} STORE_NAME;
```

Comment:
This line is the key change. The declare store macro no longer declares an instance of the store.